### ISSUE TITLE
Small typo in: Modelica.Mechanics.MultiBody.Sensors

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#130]
**Reported by adrpo on 29 Nov 2008 03:13 UTC**
Am I mistaking or the model:

```
Modelica.Mechanics.!MultiBody.Sensors.TansformAbsoluteVector
```
should be:

```
Modelica.Mechanics.!MultiBody.Sensors.TransformAbsoluteVector
```
?

Cheers, 
Adrian Pop/

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/130

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 9 Jan 2009 05:48 UTC**
I think this should be fixed for the [milestone:MSL3.0.1 3.0.1 release].

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 21 Jan 2009 09:35 UTC**
Fixed in 7cf80992038f181187ddc65a1021104607d10856. All occurrences replaced.

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 22 Jan 2009 08:35 UTC**
I completely forgot about backward compatibility (thanks Martin for reminding and reverting). 

All changes made in 7cf80992038f181187ddc65a1021104607d10856 and 9e2ce0ab10ba5d343621ab2281df6d3d6806a038 (a partly revert by Martin) are reverted in 189fb58ad7dfa2febf2f7565a7c569661c26bece.

Once the next backward compatible version is to come (probably milestone:MSL3.1) the necessary changes to fix this can be found in the attached [attachment:fixticket130.patch patch].

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 21 Jun 2009 09:58 UTC**
Will be fixed for the next MSL version that is not backward compatible and requires conversion scripts.

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 21 Jun 2009 14:30 UTC**
I'd guess the earliest non-backward compatible version can only be 3.2 but not 3.1.1

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 28 Feb 2010 17:42 UTC**
Looks like the next version with conversion script will be milestone:MSL4.0. Therefore rescheduling.

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 28 Feb 2010 17:45 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 11 Sep 2012 16:43 UTC**
FIXED in b399658cfc5ff52fd9e2b6b61ac6e4534fd9fbd8 in a backward compatible way.
TODO remove obsoleted models -> see #813

**Comment by modelica-trac-importer** [comment]
**Changelog removed by dietmarw on 11 Sep 2012 16:43 UTC**
